scalafix-core/shared/src/main/scala/scalafix/internal/rule/MissingFinal.scala

```diff
@@ -0,0 +1,62 @@
+package scalafix.internal.rule
+
+import scala.meta._
+import scalafix.lint.LintCategory
+import scalafix.patch.Patch
+import scalafix.rule.RuleCtx
+import scalafix.{SemanticRule, SemanticdbIndex}
+
+final case class MissingFinal(index: SemanticdbIndex)
+    extends SemanticRule(
+      index,
+      "MissingFinal"
+    ) {
+
+  override def description: String =
+    "Rule that checks for or adds final modifier in corresponding places"
```

Comment:
Rewrite that inserts `final` to case classes and classes that extend a sealed parent

Reply:
It inserts `final` only to case classes. I don't think inserting `final` to classes and not inserting anything to traits is a good idea since classes and traits are in the same position.